packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx

```diff
@@ -90,13 +91,21 @@ export class SuggestionsControl<T> extends BaseComponent<ISuggestionsControlProp
   }
 
   public render(): JSX.Element {
-    const { className, headerItemsProps, footerItemsProps } = this.props;
+    const { className, headerItemsProps, footerItemsProps, suggestionsAvailableAlertText } = this.props;
 
+    const screenReaderText = mergeStyles(hiddenContentStyle);
     return (
       <div className={css('ms-Suggestions', className ? className : '', styles.root)}>
         {headerItemsProps && this.renderHeaderItems()}
         {this._renderSuggestions()}
         {footerItemsProps && this.renderFooterItems()}
+        {
+          <span role="alert" aria-live="polite" className={screenReaderText}>
+            {this.state.suggestions && this.state.suggestions.length > 0 && suggestionsAvailableAlertText
```

Comment:
Suggest pulling this into a local constant (e.g. `shouldAlertSuggestionsAvailableText`)